app/javascript/components/mappers/soqlToDomain/application_preference.js

```diff
@@ -14,12 +14,14 @@ export const mapApplicationPreference = (value) => {
     preference_name:value.Preference_Name,
     person_who_claimed_name:value.Person_who_claimed_Name,
     type_of_proof:value.Type_of_proof,
+    lw_type_of_proof: value.LW_Type_of_Proof,
     opt_out:value.Opt_Out,
     lottery_status:value.Lottery_Status,
+    post_lottery_validation:value.Post_Lottery_Validation,
```

Comment:
@fddayan our style so far has been to put spaces after colons, would you have any objection to me reformatting this file to use that convention?

Reply:
not at all,  go for it